src/daemon/dlt-daemon.c

```diff
@@ -1906,78 +1914,143 @@ int dlt_daemon_local_connection_init(DltDaemon *daemon,
     return 0;
 }
 
-int dlt_daemon_local_ecu_version_init(DltDaemon *daemon, DltDaemonLocal *daemon_local, int verbose)
+static char* file_read_everything(FILE* const file, const size_t sizeLimit)
 {
-    char *version = NULL;
-    FILE *f = NULL;
-
-    PRINT_FUNCTION_VERBOSE(verbose);
```

Comment:
Why remove this macro? It is required for internal dlt logs when Verbose = 1

Reply:
I didn't remove it. Line 1914 got moved to line 2031 (below):

```
+ int dlt_daemon_local_ecu_version_init(DltDaemon *daemon, DltDaemonLocal *daemon_local, int verbose)
+ {
+    FILE *f = NULL;
+
+    PRINT_FUNCTION_VERBOSE(verbose);
```